setup-container/hyperkube/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# Creates hyperkube with CNI binaries and flannel configuration
```

Comment:
@zreigz I copied this from you pull request, temporarily until your changes are merged